ols/utils/token_handler.py

```diff
@@ -92,7 +102,7 @@ def get_available_tokens(self, prompt: str, model_config: ModelConfig) -> int:
             f"Response token limit: {response_token_limit}"
         )
 
-        prompt_token_count = len(self.text_to_tokens(prompt))
+        prompt_token_count = self._get_token_count(self.text_to_tokens(prompt))
```

Comment:
it is static method, don't have to be called via self

Reply:
yeah i was wondering about the value of a static method for this, over just making it a standalone utility function.  But i didn't want to bikeshed :)